src/main.cpp

```diff
@@ -151,6 +151,13 @@ const unsigned long intervalPressure = 100;
 unsigned long previousMillisPressure; // initialisation at the end of init()
 #endif
 
+// timing flags
+bool buffer_ready = false;
+bool display_update = false;
+bool website_update = false;
+bool mqtt_update = false;
+bool HASSIO_update = false;
+
```

Comment:
this should follow camelCase ->
displayBufferReady
displayUpdateRunning
websiteUpdateRunning
mqttUpdateRunning
hassioUpdateRunning